pkg/version/version.go

```diff
@@ -138,3 +142,23 @@ func UserAgent() string {
 
 	return "envd/" + version
 }
+
+func GetDockerVersion() (int, error) {
+
```

Comment:
Remove blank line here. And I think you can put it within builder package or in utils folder, instead of a new package here

Reply:
I change it to pkg/driver/docker in order to avoid importing extra package, PTAL